doc/sphinx/conf.py

```diff
@@ -188,13 +188,66 @@ def setup(app):
     # but is it the right place?
     'module',
     'redexpr',
+    'red_expr',
     'modpath',
     'dirpath',
     'collection',
     'term_pattern',
     'term_pattern_string',
     'command',
-    'symbol' ]]
+    'symbol',
+
+    # added for Ltac chapter
+    'tacarg',
+    'ssr_first_else',
+    'ssrseqarg',
+    'int_or_var',
+    'string_opt',
+    'ssrmmod',
+    'ssrdotac',
+    'ssrclauses',
+    'ssrortacarg',
+    'ssrdgens',
+    'query_command',
+    'int_or_var_opt',
+    'simple_tactic',
+    'reference',
+    'ssrintros_ne',
+    'STRING',
+    'lconstr',
+    'uconstr',
+    'ssrparentacarg',
+    'lconstr_pattern',
+    'component',
+    'test',
+
+    # added for gallina chapter
+    'arg',
+    'binders',
```

Comment:
`arg` and `binders` are currently defined in the Gallina chapter, so I'm failing to see why they would need to be added here.

Reply:
No, `arg` is not defined in the updated Gallina chapter; the string isn't even present.  `arg` is in the old version (and referenced by the Gallina extensions chapter).

The string `binders` appears 54 times in the file, but none of them appear to be definitions.  If you disagree, which one defines it?

Reply:
It would be good to understand what `arg` actually represented and either rename this to `arg` or use the new token everywhere that `arg` was used. Otherwise, it's kind of like introducing a regression, with tokens that used to be defined, and are not anymore but are still used.

Reply:
`arg` is only used on a couple lines in gallina-extensions.rst (text+grammar).  It looks like the grammar is incorrect (e.g. `projection` doesn't appear in `fullGrammar`).  Better to deal with this when we do that chapter.

```
     projection : `term` `.` ( `qualid` )
          : `term` `.` ( `qualid` `arg` … `arg` )
          : `term` `.` ( @`qualid` `term` … `term` )
```